src/main/scala/it/pagopa/interop/purposeprocess/service/RiskAnalysisService.scala

```diff
@@ -7,12 +7,30 @@ import it.pagopa.interop.tenantmanagement.client.model.TenantKind.{GSP, PA, PRIV
 import scala.io.Source
 import spray.json._
 
-object RiskAnalysisService {
+trait RiskAnalysisServiceSupplier {
+  def get(): RiskAnalysisService
+}
+
+object RiskAnalysisServiceSupplier extends RiskAnalysisServiceSupplier {
+  override def get(): RiskAnalysisService = new RiskAnalysisServiceImpl
+}
+
+trait RiskAnalysisService {
+  def riskAnalysisForms(): Map[TenantKind, Map[String, RiskAnalysisFormConfig]]
+  def loadRiskAnalysisFormConfig(resourcePath: String): RiskAnalysisFormConfig
+}
+class RiskAnalysisServiceImpl extends RiskAnalysisService {
+  override def riskAnalysisForms(): Map[TenantKind, Map[String, RiskAnalysisFormConfig]] =
+    RiskAnalysisServiceImpl.riskAnalysisForms
+  override def loadRiskAnalysisFormConfig(resourcePath: String): RiskAnalysisFormConfig  =
+    RiskAnalysisServiceImpl.loadRiskAnalysisFormConfig(resourcePath)
+}
```

Comment:
Is this really required? We already have the `RiskAnalysisServiceImpl`, which is a singleton and does not require the creation of a new instance
```suggestion
```